de/tests/test_factorization.py

```diff
@@ -4,7 +4,7 @@
 import pytest
 import numpy as np
 from scipy.special import expit
-from ..factorization import factorizeEstimate, alpha
+from ..factorization import MatrixSubtraction, cellLineComparision, factorizeEstimate, alpha, cellLineFactorization
```

Comment:
There is a spelling error here.. 
`cellLineComparison` is written `cellLineComparision`
I think this will fix your test error